src/Program.cs

```diff
@@ -14,12 +14,11 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            BuildWebHost(args).Run();
+            CreateWebHostBuilder(args).Build().Run();
         }
 
-        public static IWebHost BuildWebHost(string[] args) =>
+        public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
             WebHost.CreateDefaultBuilder(args)
-                .UseStartup<Startup>()
-                .Build();
```

Comment:
Why is Build() removed?

Reply:
It's on line 17 now. See Microsofts guidance - https://docs.microsoft.com/en-us/aspnet/core/migration/20_21?view=aspnetcore-2.1#changes-to-take-advantage-of-the-new-code-based-idioms-that-are-recommended-in-aspnet-core-21